root/components/Table.js

```diff
@@ -29,7 +27,9 @@ const renderTableHeaderRow = (headerGroup) => (
 );
 
 const renderTableCell = (cell) => (
-  <td {...cell.getCellProps({className: cell.column.className})}>
+  <td
+    {...cell.getCellProps(cell.column.cellProps)}
+  >
```

Comment:
Some line breaks were added inside the tag here, but it actually makes the original line shorter, so probably not needed